cmd/storj-sim/console.go

```diff
@@ -80,7 +80,7 @@ func generateActivationKey(userID uuid.UUID, email string, createdAt time.Time)
 	return token.String(), nil
 }
 
-func addExampleProjectWithKey(key *string, address string) error {
+func addExampleProjectWithKey(key *string, activationAddress, address string) error {
```

Comment:
does storj-sim need to adjusted as well?

Reply:
@egonelbre we need this changes for integration tests